docs/source/release/v6.3.0/33203_framework.rst

```diff
@@ -0,0 +1,9 @@
+SliceViewer
```

Comment:
Quick question, I think SliceViewer release notes were going into workbench and not framework? As it's all GUI then I think that makes sense?

Reply:
I put it in Framework because it has a sliceviewer section heading, but I agree workbench makes more sense. Do the templates need updating? @sf1919 do you want me to change the title of the release notes file?

Reply:
No its ok. I'll make a note to put it in the right place when the time comes.

@martyngigg does the auto generation script for release notes need updating? At the moment it only puts SliceViewer in Framework and not at all in Workbench - https://github.com/mantidproject/mantid/blob/main/docs/source/release/v6.4.0/framework.rst

Reply:
Ah, thanks for raising that @sf1919. Yes I think that should be changed as I don't think it makes sense to have them in the framework.

Reply:
Yes - just merge it in. I've made a note. Seems silly to hold it up any further.

Reply:
Perfect - thanks.